libs/SalesforceAnalytics/build.gradle

```diff
@@ -52,3 +53,68 @@ afterEvaluate {
   // from timing out since executing tests are printed to the console.
   tasks.matching { it.name ==~ /^connected.*AndroidTest$/ }*.logging*.setLevel(LogLevel.INFO)
 }
+
+install {
```

Comment:
Packaging and uploading multiple libraries/artifacts from a single repo is very tricky. Hence, even though a lot of this seems repeated, we need them in individual `Gradle` files. You'll notice `artifactId` is different, and there are some other differences too.